pom.xml

```
@@ -6,7 +6,7 @@
     <groupId>org.moltimate.backend</groupId>
     <artifactId>moltimate-backend</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <packaging>war</packaging>
+    <packaging>jar</packaging>
```

```suggestion
    <packaging>war</packaging>
```
This needs to be war for deployments to GCP